rubocop/cop/lint/last_keyword_argument.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop only works if there are files from deprecation_toolkit. You can
+      # generate these files by:
+      #
+      # 1. Running specs with RECORD_DEPRECATIONS=1
+      # 1. Downloading the complete set of deprecations/ files from a CI
+      # pipeline (see https://gitlab.com/gitlab-org/gitlab/-/merge_requests/47720)
+      class LastKeywordArgument < RuboCop::Cop::Base
+        extend RuboCop::Cop::AutoCorrector
+
+        MSG = 'Using the last argument as keyword parameters is deprecated'
+
+        DEPRECATIONS_GLOB = File.expand_path('../../../deprecations/**/*.yml', __dir__)
+        KEYWORD_DEPRECATION_STR = 'maybe ** should be added to the call'
+
+        def on_send(node)
```

Comment:
[pronto-rubocop warning] Metrics/CyclomaticComplexity: Cyclomatic complexity for on_send is too high. [11/7]